src/Components/ListOfArticles.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import axios from 'axios';
-import { Grid, Container } from 'semantic-ui-react'
+import { Grid, Container, Card, Image, Icon } from 'semantic-ui-react'
```

Comment:
Strange.. I thought these changes was already merged in the PT of "Styling landingpage" since I have them when pull upstream? But if there are no conflicts then I guess no worries :)

Reply:
This story was started 4 days ago. The styling was merged yesterday.